developer-motivator/functions/index.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
```

Comment:
Need to add an Apache 2.0 Google licence header at the top of the file (see other similar files)